src/symposium-app/symposium-card/symposium-card.html

```diff
@@ -25,19 +23,19 @@
       }
     </style>
 
-    <paper-card>
-      <div class="card-content">
-        <content></content>
-      </div>
+    <paper-card heading={{heading}}>
```

Comment:
Since this element is very similar to the lancie one, lets mark it for an external wisvch element